pkg/sdk/go/keys.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const keysEndpoint = "keys"
 
-func (sdk mfSDK) Issue(token string, k Key) (issueKeyRes, error) {
+func (sdk mfSDK) Issue(token string, k KeyReq) (issueKeyRes, error) {
```

Comment:
Why is request called `KeyReq`, and the response `issueKeyRes`? Can we please align these two for consistence?